code/game/supplyshuttle.dm

```diff
@@ -421,13 +501,7 @@ var/datum/controller/supply/supply_controller = new()
 
 //To stop things being sent to centcomm which should not be sent to centcomm. Recursively checks for these types.
 /datum/controller/supply/proc/forbidden_atoms_check(atom/A)
-	if(istype(A,/mob/living))
-		return 1
-	if(istype(A,/obj/item/disk/nuclear))
-		return 1
-	if(istype(A,/obj/item/device/radio/beacon))
-		return 1
-	if(istype(A,/obj/item/stack/sheet/mineral/phoron))
+	if(istype(A,/mob/living) && !black_market_enabled)
 		return 1
```

Comment:
![image](https://user-images.githubusercontent.com/55142896/213929838-eb01b857-efd4-4af6-acac-11226402c8b5.png)

Reply:
set it to true dumbass not 1 i HATE 1